common/components/message/_message.scss

```diff
@@ -22,6 +22,10 @@ $_message-border-width: 5px;
     margin-top: 0;
   }
 
+  p {
```

Comment:
Is there not a govuk mixin for this? Why 600px?

Reply:
So that was to set the [optimal reading width](https://www.freshconsulting.com/uiux-principle-46-text-box-width-should-help-users-read/#:~:text=700%20pixels%20wide%20is%20a,(2.33%20inches)%20in%20width.) as the text shouldn't really stretch the whole width of the page.

I'll have a check see if there's something in GOV.UK Frontend but I couldn't see anything...

Reply:
I'll extract it to something more useful in our app code :)